### PR TITLE
Keyboard support for visit table

### PR DESCRIPTION
**Pull request by Johngtka**
Checklist:
- [X] `working left and right arrow switching on visit table`
- [X] `switching is avaliable by pc keyboard`

ezgif com-video-to-gif

### CODE REVIEW

**Comment by Johngtka** [comment]
S&M commit poposal:
`feat: visit table switching by left and right arrow from keyboard`

File: src/app/visit/visit.component.ts [hunk 1]
```diff
@@ -56,4 +54,12 @@ export class VisitComponent implements OnInit {
     selectPatient(patientSelected: Patient) {
         this.patient = patientSelected;
     }
+    @HostListener('document:keydown', ['$event'])
+    handleKeyboardEvent(event: KeyboardEvent) {
+        if (event.key === 'ArrowRight') {
+            this.paginator.nextPage();
```
Comment (macrog) [inline src/app/visit/visit.component.ts:60]: I would add here a check if there is a next or previous page

Comment (Johngtka) [reply src/app/visit/visit.component.ts:60]: done but explain a this condition:
`if (event.key === 'ArrowRight' && this.paginator.hasNextPage())`

**Review by macrog — APPROVED** [review]
Code looks good 👍

**Pull request merged**